src/foundry/client/ui/editor.d.ts

```diff
@@ -1,5 +1,5 @@
 import type { ConfiguredDocumentClassForName } from '../../../types/helperTypes';
```

Comment:
I think this needs to be extended to also support prosemirror. Have a look at #2095 's `FormApplication#activateEditor`. It effectively returns the value of `TextEditor.create` and is marked to return either `tinymce.Editor` or one of prosemirror's `EditorView`s.

Reply:
Right, going to make this depend on #2104 then, as I pull in the types for it there.